webview/src/experiments/components/App.tsx

```diff
@@ -1,19 +1,19 @@
-import React, { useCallback } from 'react'
-import { useDispatch } from 'react-redux'
+import { TableData } from 'dvc/src/experiments/webview/contract'
```

Comment:
During the creation of this PR I moved the reducers in and out of this file. Rather than struggle to revert the exact order I resorted to using VS Code's "Organize Imports":

![image](https://github.com/iterative/vscode-dvc/assets/37993418/43c64fec-b94f-4e88-b374-fd371e5113eb)

Rather than fight the new order I think we should start using this (means I have to think less to make things consistent).

Reply:
I used to do this, but it would clash with some linting we have. There might be a config that was needed.